Sources/KlaviyoSDKDependencies/ComposableArchitecture/RootStore.swift

```diff
@@ -0,0 +1,144 @@
+import Combine
+import KlaviyoSDKDependencies
```

Comment:
```suggestion
```
we shouldn't need this because it's already part of the `KlaviyoSDKDependencies` package